packages/framework-core/src/services/graphql/graphql-query-generator.ts

```diff
@@ -29,32 +38,27 @@ export class GraphQLQueryGenerator {
   private generatedOperationEnumsByTypeName: Record<string, GraphQLEnumType> = {}
 
   public constructor(
+    private readonly config: BoosterConfig,
     private readonly targetTypes: TargetTypesMap,
     private readonly typeInformer: GraphQLTypeInformer,
     private readonly byIDResolverBuilder: ResolverBuilder,
-    private readonly filterResolverBuilder: ResolverBuilder
+    private readonly filterResolverBuilder: ResolverBuilder,
+    private readonly eventsResolver: GraphQLFieldResolver<unknown, GraphQLResolverContext, any>
   ) {}
 
   public generate(): GraphQLObjectType {
     const byIDQueries = this.generateByIDQueries()
     const filterQueries = this.generateFilterQueries()
-    const fields = {...byIDQueries, ...filterQueries}
-    if (Object.keys(fields).length === 0) {
-      return new GraphQLObjectType({
-        name: 'Query',
-        fields: {
-          NoQueriesDefined: { type: GraphQLString }
-        }
-      })
-    }
```

Comment:
Now ther will always be queries defined: the ones to read events

Reply:
There can be conflicts with the new GraphQL schema branch, just to have it in mind